src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.TargetFrameworkInference.targets

```diff
@@ -88,6 +88,11 @@ Copyright (c) .NET Foundation. All rights reserved.
     <_UnsupportedTargetFrameworkError>true</_UnsupportedTargetFrameworkError>
   </PropertyGroup>
 
+  <!--Default MinimumOSPlatform to TargetPlatformVersion-->
+  <PropertyGroup Condition="'$(MinimumOSPlatform)' == '' and '$(TargetPlatformVersion)' != ''">
```

Comment:
Should it be MinimumPlatformVersion to more closely match TargetPlatformVersion, especially since it defaults to it? Isn't "OS" and "Platform" the same in this context, so the naming is redundant, or am I overlooking something?

Reply:
Agreed `OSPlatform` is inherently redundant, but it aligns with the existing struct that is used in the API for runtime checks.